Jenkinsfile

```diff
@@ -183,9 +183,16 @@ def runBuild(abiFilter, instrumentationTestTarget) {
     },
     'Static code analysis' : {
       try {
-        gradle('realm', "findbugs ${abiFilter}") // FIXME Renable pmd and checkstyle
+        gradle('realm', "spotbugsMain ${abiFilter}") // FIXME Renable pmd and checkstyle
```

Comment:
There also seem to be spotbugsBase and spotBugsObjectServer. Not 100% sure what `Main` covers?

Reply:
Ah sorry. Missed the change where you configure this as its own target 🤦 👍

Reply:
It doesnt allow me to set the "spotbugs" name to the task.